scripts/repeatIT.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+targetTest=${1:-ExpositionIT}
+
+logFile=/tmp/mvn-${targetTest}-$(date +"%d-%m-%Y_%H:%M").log
+
+mvn clean install -DskipTests=true
+
+for COUNT in {1..10} ; do
+    echo "=============== RUN ${COUNT} STARTING =============== " | tee --append "${logFile}"
+    mvn verify -Dit.test=${targetTest} -pl :integrationtests >> "${logFile}"
```

Comment:
The nuances of output redirection in bash evade me, but something about this doesn't seem to forward stderr to the log file. Perhaps we could alter this to `mvn verify -Dit.test=${targetTest} -pl :integrationtests 2&>1 | tee -a "${logFile}"` ?

Reply:
a good point, not sure we need std errbut lets do it anyway.

Reply:
fixed in e5ba3bf9e63aadfb707058f8d3ba56a63f596b76